src/frontend/AuthProvider.jsx

```diff
@@ -83,29 +208,30 @@ export const KindeProvider = ({children}) => {
     try {
       const tokens = await tokenFetcher(setupUrl);
 
-      const user = {
-        id: tokens.id_token.sub,
-        name: tokens.id_token.name,
-        given_name: tokens.id_token.given_name,
-        family_name: tokens.id_token.family_name,
-        updated_at: tokens.id_token.updated_at,
-        email: tokens.id_token.email,
-        picture: tokens.id_token.picture
-      };
+      const {
+        user,
+        permissions,
+        organization,
+        userOrganizations,
+        featureFlags,
+        accessToken,
+        idToken
+      } = tokens;
+
+      const getAccessToken = () => accessToken;
+      const getIdToken = () => accessToken;
```

Comment:
@peterphanouvong is this right? Seems like it should return the idToken not accessToken